compat/src/suspense.js

```diff
@@ -22,6 +22,15 @@ options._catchError = function(error, newVNode, oldVNode) {
 	oldCatchError(error, newVNode, oldVNode);
 };
 
+const oldUnmount = options.unmount;
```

Comment:
You are never calling oldUnmount, this could lead to a plugin chain failing.

Reply:
oops good catch 🙈